### PR TITLE
Create lss-manager-v3-nl.dev.js

### PR DESCRIPTION
**Pull request by DMCEM4**
Dutch translation added

### CODE REVIEW

File: lss-manager-v3/lss-manager-v3-nl.dev.js [hunk 1]
```diff
@@ -303,11 +315,13 @@ lssm.Module = {
     MissionOut: {
         name: {
             de: 'MissionOut'
```
Comment (lostdesign) [inline lss-manager-v3/lss-manager-v3-nl.dev.js]: missing `,`

File: lss-manager-v3/lss-manager-v3-nl.dev.js [hunk 2]
```diff
@@ -275,6 +285,7 @@ lssm.Module = {
         description: {
             de: 'Karte im linken Bereich auf 100% Höhe. Rechts davon die Einsätze auf voller breite - alle Einsatzarten werden in jeweils einer Spalte dargestellt. Darunter Gebäude, Chat und Funk.',
             en: 'Map with 100% height on the left side. Next to it the calls, each category in its own column. Below that the buildings, chat and radio.'
```
Comment (lostdesign) [inline lss-manager-v3/lss-manager-v3-nl.dev.js]: missing `,`

File: lss-manager-v3/lss-manager-v3-nl.dev.js [hunk 3]
```diff
@@ -201,11 +206,13 @@ lssm.Module = {
         name: {
             de: 'Keyboard Alarmierung',
             en: 'Callview control'
```
Comment (lostdesign) [inline lss-manager-v3/lss-manager-v3-nl.dev.js]: missing `,`

File: lss-manager-v3/lss-manager-v3-nl.dev.js [hunk 4]
```diff
@@ -336,11 +352,13 @@ lssm.Module = {
     Notification_Alert: {
         name: {
             de: 'Notification Alert'
```
Comment (lostdesign) [inline lss-manager-v3/lss-manager-v3-nl.dev.js]: missing `,`

File: lss-manager-v3/lss-manager-v3-nl.dev.js [hunk 5]
```diff
@@ -357,6 +375,7 @@ lssm.Module = {
         description: {
             de: 'Neues Design für die Oberfläche',
             en: 'New design for the game.'
```
Comment (lostdesign) [inline lss-manager-v3/lss-manager-v3-nl.dev.js]: missing `,`

File: lss-manager-v3/lss-manager-v3-nl.dev.js [hunk 6]
```diff
@@ -420,12 +445,14 @@ lssm.Module = {
     tagMissions: {
         name: {
             de: 'Einsätze markieren',
-            en: 'Mark calls'
+            en: 'Mark calls',
+            nl: 'Meldingen markeren.
```
Comment (lostdesign) [inline lss-manager-v3/lss-manager-v3-nl.dev.js]: missing `,` and `'`

**Comment by DMCEM4** [comment]
Fixed all the , and '

**Comment by lostdesign** [comment]
We cannot merge into master. Please reopen and make the PR for the dev branch.

**Pull request closed without merging**